core/src/test/resources/Align/DataTypes/DataTypes-DataTypes.jsd

```diff
@@ -202,9 +205,7 @@
             },
             "PatternString":{
                 "type":"string",
-                "pattern":[
-                    "J|N"
-                ]
+                "pattern":"J|N"
```

Comment:
Dit klopt volgens mij niet?

Reply:
Ik heb vanochtend een call gehad met Jeroen waarin een array met een single value volgens de JSON schema validatie in XML Spy niet geaccepteerd werd.

Daarom deze change.